From 77bd687bac833055b82e2bb6137e2be5fcb77d21 Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Tue, 4 Aug 2020 12:05:56 +0200
Subject: [PATCH] acp: fix size of array

---
 spa/plugins/alsa/acp/acp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/spa/plugins/alsa/acp/acp.c b/spa/plugins/alsa/acp/acp.c
index cd33296d..943ed6fb 100644
--- a/spa/plugins/alsa/acp/acp.c
+++ b/spa/plugins/alsa/acp/acp.c
@@ -317,7 +317,7 @@ static int report_jack_state(snd_mixer_elem_t *melem, unsigned int mask)
     pa_log_debug("Jack '%s' is now %s", pa_strnull(snd_hctl_elem_get_name(elem)),
 		    plugged_in ? "plugged in" : "unplugged");
 
-    size = sizeof(struct temp_port_avail) * pa_hashmap_size(impl->jacks)+1;
+    size = sizeof(struct temp_port_avail) * (pa_hashmap_size(impl->jacks)+1);
     tports = tp = alloca(size);
     memset(tports, 0, size);
 
-- 
2.26.2

