From 78fa6b851fc5f3ed669cb1487aee315ad6a45389 Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Fri, 24 Nov 2017 17:23:40 +0100
Subject: [PATCH] Only define memfd_create when not already defined

---
 src/pipewire/memfd-wrappers.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/pipewire/memfd-wrappers.h b/src/pipewire/memfd-wrappers.h
index 05958fe..75ed628 100644
--- a/src/pipewire/memfd-wrappers.h
+++ b/src/pipewire/memfd-wrappers.h
@@ -20,6 +20,7 @@
 #include <sys/syscall.h>
 #include <fcntl.h>
 
+#ifndef memfd_create
 /*
  * No glibc wrappers exist for memfd_create(2), so provide our own.
  *
@@ -32,6 +33,7 @@ static inline int
 memfd_create(const char *name, unsigned int flags) {
 	return syscall(SYS_memfd_create, name, flags);
 }
+#endif
 
 /* memfd_create(2) flags */
 
-- 
2.14.3

