From bd6d72be8f7ceed231661c01b5ad21d8fde9c705 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Nils=20Tonn=C3=A4tt?= <nils.tonnaett@posteo.de>
Date: Sun, 18 Apr 2021 15:14:30 +0200
Subject: [PATCH 2/4] Install jack headers

---
 pipewire-jack/meson.build | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pipewire-jack/meson.build b/pipewire-jack/meson.build
index 5968c84d..c295e45e 100644
--- a/pipewire-jack/meson.build
+++ b/pipewire-jack/meson.build
@@ -1,2 +1,5 @@
 jack_inc = include_directories('.')
+if get_option('jack-devel').enabled()
+  install_subdir('jack', install_dir: get_option('includedir'), strip_directory: false)
+endif
 subdir('src')
-- 
2.31.1

