From 4bb859fb82d4d8128e146937289d560a8efa0fc3 Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Sat, 31 Oct 2020 21:20:39 +0100
Subject: [PATCH 2/3] pulse-server: use name if description not set

---
 src/modules/module-protocol-pulse/collect.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/modules/module-protocol-pulse/collect.c b/src/modules/module-protocol-pulse/collect.c
index 130af355..9be5a8ae 100644
--- a/src/modules/module-protocol-pulse/collect.c
+++ b/src/modules/module-protocol-pulse/collect.c
@@ -244,6 +244,8 @@ static uint32_t collect_profile_info(struct pw_manager_object *card, struct card
 				SPA_PARAM_PROFILE_classes,  SPA_POD_OPT_Pod(&classes)) < 0) {
 			continue;
 		}
+		if (pi->description == NULL)
+			pi->description = pi->name;
 		if (pi->id == card_info->active_profile)
 			card_info->active_profile_name = pi->name;
 
@@ -443,6 +445,8 @@ static uint32_t collect_port_info(struct pw_manager_object *card, struct card_in
 				SPA_PARAM_ROUTE_profiles,  SPA_POD_OPT_Pod(&profiles)) < 0)
 			continue;
 
+		if (pi->description == NULL)
+			pi->description = pi->name;
 		if (devices)
 			pi->devices = spa_pod_get_array(devices, &pi->n_devices);
 		if (profiles)
-- 
2.28.0

