From 01951fd278fece5bd7ef1d504788125a2b8e1fa9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Nils=20Tonn=C3=A4tt?= <nils.tonnaett@posteo.de>
Date: Sun, 18 Apr 2021 15:47:58 +0200
Subject: [PATCH 3/4] Generate JACK pkgconfig file

---
 pipewire-jack/src/meson.build | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/pipewire-jack/src/meson.build b/pipewire-jack/src/meson.build
index 148db282..32d37c48 100644
--- a/pipewire-jack/src/meson.build
+++ b/pipewire-jack/src/meson.build
@@ -47,6 +47,15 @@ pipewire_jack = shared_library('jack',
     install_dir : libjack_path,
 )
 
+if get_option('jack-devel').enabled()
+  pkgconfig.generate(filebase : 'jack',
+  libraries : pipewire_jack,
+  name : 'jack',
+  description : 'PipeWire JACK API',
+  version : '1.9.17',
+  extra_cflags : '-D_REENTRANT')
+endif
+
 pipewire_jackserver = shared_library('jackserver',
     pipewire_jackserver_sources,
     soversion : soversion,
-- 
2.31.1

