From cde4d53c867b23d97ae37dd9eb518d0a62170c44 Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Tue, 10 Aug 2021 11:14:15 +0200
Subject: [PATCH 5/6] alsa: free alibpref after use

---
 spa/plugins/alsa/alsa-pcm.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/spa/plugins/alsa/alsa-pcm.c b/spa/plugins/alsa/alsa-pcm.c
index dfffcb478..30fbbba6e 100644
--- a/spa/plugins/alsa/alsa-pcm.c
+++ b/spa/plugins/alsa/alsa-pcm.c
@@ -52,6 +52,7 @@ int spa_alsa_init(struct state *state)
 			spa_scnprintf(name, sizeof(name), "%s%s", alibpref,
 					state->props.device);
 			strcpy(state->props.device, name);
+			free((void*)alibpref);
 		}
 	}
 	return 0;
-- 
2.31.1

