From ca5f08e701b9a8f271637c51040fdfb8e8daf0f5 Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Tue, 27 Apr 2021 15:38:31 +0200
Subject: [PATCH 08/10] conf: remove X bit from config files

Fixes #1107
---
 src/pipewire/conf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pipewire/conf.c b/src/pipewire/conf.c
index 50e0b0f9..24b15ffb 100644
--- a/src/pipewire/conf.c
+++ b/src/pipewire/conf.c
@@ -193,7 +193,7 @@ int pw_conf_save_state(const char *prefix, const char *name, struct pw_propertie
 
 	tmp_name = alloca(strlen(name)+5);
 	sprintf(tmp_name, "%s.tmp", name);
-	if ((fd = openat(sfd, tmp_name,  O_CLOEXEC | O_CREAT | O_WRONLY | O_TRUNC, 0700)) < 0) {
+	if ((fd = openat(sfd, tmp_name,  O_CLOEXEC | O_CREAT | O_WRONLY | O_TRUNC, 0600)) < 0) {
 		pw_log_error("can't open file '%s': %m", tmp_name);
 		res = -errno;
 		goto error;
-- 
2.30.2

